File: 2015/8. Matchsticks/1.Solution.py
ans = 0

for i in range(300):
    s = input()
    x = len(s)
    y = 0

    i = 1
    while s[i] != '"':
        if s[i] == '\\':
            if s[i+1] == '\\' or s[i+1] == '"':
                i += 1
            else:
                i += 3
        y += 1
        i += 1
    ans += x - y

print(ans)

File: 2016/3. Squares With Three Sides/1.Solution.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int ans = 0;
    int x, y, z;
    for (int i = 0; i < 1734; i++) { // no of lines of input
        cin >> x >> y >> z;
        if (x + y > z and x + z > y and z + y > x) ans++;
    }
    cout << ans;
}
